cpp/modmesh/CMakeLists.txt

```diff
@@ -8,6 +8,19 @@ if (BUILD_QT)
     set(CMAKE_AUTORCC ON)
     set(CMAKE_AUTOUIC ON)
 
+    find_package(Qt6 QUIET)
+    if(NOT Qt6_FOUND)
+        if(MSVC)
+            message(FATAL_ERROR  "Qt6 Not Found.\nTry to add `C:\\Qt\\6.x.y\\msvcYYYY_64\\lib\\cmake\\Qt` into \"Path\" of \"Enviroment Variables\"?")
+        endif()
+        if(APPLE)
+            message(FATAL_ERROR  "Qt6 Not Found.\nTry `CMAKE_PREFIX_PATH=/path/to/qt/6.x.y/macos`?")
+        endif()
+        if(UNIX AND NOT APPLE)
+            message(FATAL_ERROR  "Qt6 Not Found.\nTry `cmake -DCMAKE_PREFIX_PATH=/path/to/Qt/6.y.z/gcc_64/lib/cmake`?")
```

Comment:
I think it should follow the `6.x.y` convention as above.  Or the above two may change to `6.y.z`.  I am fine either way, but they should be consistent.

Reply:
sure